### DISCUSSION TITLE
Dịch hay giữ nguyên "clone", "fork", "repo", "pull requests", "issues"

### DISCUSSION THREAD

**Discussion by trgiangdo** [d2l-ai/d2l-vi#3022] [Terminology]
Các từ này xuất hiện nhiều trong hướng dẫn sử dụng Github trong Hướng dẫn đóng góp cho quyển sách.

Em hiện đang dịch:
`fork` -> `tạo rẽ nhánh` hoặc `rẽ nhánh`
`repo` hoặc `repository` -> `kho chứa`
`clone` -> `tạo bản sao`, tuy nhiên có sự trùng lặp so với `fork` (tạo bản sao về username và về local).
`pull requests` và `issues` hiện chưa có nghĩ ra từ nào thích hợp.

Mọi người cho ý kiến nhé.

**cc: @minhduc0711 @rootonchair @davidnvq @duythanhvn @cuongvng @lkhphuc**

**Comment by thanhcsf** [comment]
Nhóm từ này nên giữ nguyên, thật ra là có thể dịch và để chú thích nhỉ...

> `clone` -> `tạo bản sao`, tuy nhiên có sự trùng lặp so với `fork` (tạo bản sao về username và về local).

Fork thì trên github, clone thì về local. Không trùng lặp đâu em.

**Reply by lkhphuc** [reply]
Các từ này mình xài tiếng anh quen rồi, không biết mọi người sao có nên dịch không.
Nếu dịch thì `issue: vấn đề`, `PR: yêu cầu gộp`
Từ `fork: ....` của Github khá khó dịch, `rẽ nhánh` thì sẽ bị nhầm với `branch` trong git, `tạo bản sao` thì nhầm với `clone`, còn nói đầy đủ `sao chép sang một (repo) kho chứa khác` thì dài dong.

**Reply by cuongvng** [reply]
Fork: dĩa nhánh 
Em nghĩ từ chế ra này gợi hình :v nghe vẫn hiểu hiểu là ra nhánh khác. 'dĩa' gợi ra 'folk'

**Reply by rootonchair** [reply]
Theo mình thấy thì mất từ bên IT không có gốc gác rõ ràng và dễ suy diễn như những từ trong Toán vậy nên opinion của mình là không dịch

**Reply by lkhphuc** [reply]
Mình cũng vote không dịch. Từ này chắc bên IT xài nhiều rồi.

**Reply by thanhcsf** [reply]
Vậy điểm này chốt giữ nguyên anh em nhé.

**Reply by LongNguyen1984** [reply]
Mình nghĩ không dịch nhưng nên có phần giải nghĩa ở đâu đó chứ nhỉ?

**Reply by rootonchair** [reply]
@LongNguyen1984 đúng anh, điều này thì có thể giải thích trong bản dịch.
